src/pages/task/ActionButtons.tsx

```diff
@@ -323,7 +323,7 @@ export const ActionButtons: React.VFC<Props> = ({
             setIsVisibleModal(true);
           }}
         >
-          Notify Me
+          Notify me
```

Comment:
there's another `"Notify Me"` for the patch buttons, did you want to update that too?